esp/bindings/http/platform/httptransport.cpp

```diff
@@ -798,6 +798,49 @@ void CHttpMessage::logSOAPMessage(const char* message, const char* prefix)
     return;
 }
 
+static const char* POST_METHOD_STR = "POST ";
+static bool skipLogContent(const char* httpHeader)
+{
+    if (!startsWith(httpHeader, POST_METHOD_STR))
+        return false;
+
+    const char* servicePtr = httpHeader + 5;
+    if (isEmptyString(servicePtr) || (servicePtr[0] != '/'))
```

Comment:
trivial.  The first test is redundant - it cannot be null, and if it is '\0' that will not match '/'

Reply:
Will remove the first test.